packages/suite-web/static/js/browserDetection.js

```diff
@@ -0,0 +1,50 @@
+
+    // this should match browserslist config
+    var supportedBrowsers = [
+        {
+            name: 'Chrome',
+            version: 59
+        },
+        {
+            name: 'Firefox',
+            version: 54
+        }
+    ];
+    var parser = new UAParser();
+    var result = parser.getResult();
+    var isSupported = supportedBrowsers.filter(function(browser) {
```

Comment:
array.filter is not supported on < IE 9

Reply:
I added extension of Array prototype if there is no filter method in Array.prototype